### PR TITLE
Upgrade badges in README (PYPI)

### PR DESCRIPTION
**Pull request by webknjaz**
##### SUMMARY
Warehouse (new PYPI) [is going to fully replace Cheeseshop](https://github.com/pypa/warehouse/milestone/1) soon, thus badge link needs updating.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
README.rst

##### ANSIBLE VERSION
```
devel
```

##### ADDITIONAL INFORMATION
‒

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -99,6 +99,8 @@ GNU General Public License v3.0
 See `COPYING <COPYING>`_ to see the full text.
 
 .. |PyPI version| image:: https://img.shields.io/pypi/v/ansible.svg
-   :target: https://pypi.python.org/pypi/ansible
+   :target: https://pypi.org/project/ansible
+.. |Docs badge| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg
```
Comment (gundalow) [inline README.rst]: This seems to be a static image, is that correct?

Comment (gundalow) [reply README.rst]: Oh, just so we have a more visible link to the docs?

Comment (abadger) [reply README.rst]: Yeah.  A more visible link that fits in with the theme of the other badges.

**Comment by abadger** [comment]
Discussed on chat.  We +1'd the docs badge but asked that the update to the pypi URL wait until they've actually left beta so we don't direct users to a page with the beta warning.

**Comment by webknjaz** [comment]
Did #38543 for badge. This one can be postponed until after they unbeta warehouse

**Comment by webknjaz** [comment]
https://twitter.com/di_codes/status/985896605294448640?s=19
No "Beta" banner anymore!
@abadger now we can finally merge this :)

**Comment by webknjaz** [comment]
Backported as 867bc89

**Pull request merged**